google/cloud/storage/emulator/utils/common.py

```diff
@@ -330,3 +338,87 @@ def enforce_patch_override(request):
         and request.headers.get("X-Http-Method-Override", "") != "PATCH"
     ):
         utils.error.notallowed(context=None)
+
+
+def __extract_data(data):
+    if isinstance(data, FlaskResponse):
+        return data.get_data()
+    if isinstance(data, dict):
+        return json.dumps(data)
+    return data
+
+
+def __wrap_in_response_instance(data, to_wrap):
+    if isinstance(data, FlaskResponse):
+        data.set_data(to_wrap)
+        return data
+    return FlaskResponse(to_wrap)
+
+
+def __get_streamer_response_fn(database, method, conn, test_id):
+    def response_handler(data):
+        def streamer():
+            database.dequeue_next_instruction(test_id, method)
+            d = __extract_data(data)
+            chunk_size = 4
```

Comment:
seems like `chunk_size` and `10` should be maybe parameters?  With default values?